Classes/Extension/ExtensionSetupRenderer.php

```diff
@@ -0,0 +1,122 @@
+<?php
+namespace Helhum\Typo3Console\Extension;
+
+/*
+ * This file is part of the TYPO3 Console project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read
+ * LICENSE file that was distributed with this source code.
+ *
+ */
+
+use Helhum\Typo3Console\Database\Schema\SchemaUpdateResult;
+use Helhum\Typo3Console\Database\Schema\SchemaUpdateType;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Extbase\SignalSlot\Dispatcher;
+use TYPO3\CMS\Extensionmanager\Utility\InstallUtility;
+
+/**
+ * Class ExtensionSetup
+ */
+class ExtensionSetupRenderer
+{
+    /**
+     * @var \Helhum\Typo3Console\Mvc\Cli\ConsoleOutput
+     * @inject
+     */
+    protected $output;
+
+    /**
+     * @var \Helhum\Typo3Console\Database\Schema\SchemaUpdateResultRenderer
+     * @inject
+     */
+    protected $schemaUpdateResultRenderer;
+
+    /**
+     * Mapping between slot name and method name at this object
+     *
+     * @var array
+     */
+    protected $slotToMethodMapping = [
+        'afterExtensionFileImport' => 'outputExtensionFileImport',
+        'afterExtensionStaticSqlImport' => 'outputImportedStaticData',
+        'afterExtensionT3DImport' => 'outputExtensionDataImport',
+    ];
+
+    /**
+     * Render schema result
+     *
+     * @param SchemaUpdateResult $result
+     */
+    public function renderSchemaResult($result)
+    {
+        if ($result->hasPerformedUpdates()) {
+            $this->output->outputLine('<info>The following database schema updates were performed:</info>');
+            $this->schemaUpdateResultRenderer->render($result, $this->output, true);
+        } else {
+            $this->output->outputLine(
+                '<info>No schema updates were performed for update types:%s</info>',
+                [
+                    PHP_EOL . '"' . implode('", "', SchemaUpdateType::expandSchemaUpdateTypes(['safe'])) . '"',
+                ]
+            );
+        }
+    }
+
+    /**
+     * Output at cli context imported static database content.
+     *
+     * @param string $pathToStaticSqlFile
+     * @return void
+     */
+    public function outputImportedStaticData($pathToStaticSqlFile)
+    {
+        // Output content of $pathToStaticSqlFile at cli context
+        $absolutePath = PATH_site . $pathToStaticSqlFile;
+        if (file_exists($absolutePath)) {
+            $this->output->outputFormatted('<info>Import content of file "%s" into database.</info>', [$pathToStaticSqlFile]);
+            $this->output->outputFormatted(file_get_contents($absolutePath), [], 2);
```

Comment:
The benefit is to see what happens at an automatic deployment.
It would be better to read the exact database queries that was executed by TYPO3. But this is not possible at the moment.